### ISSUE TITLE
incoprorate PNW streamgage catalog

### ISSUE THREAD

**Issue by ksonda** [internetofwater/ref_gages#25]
@ewiggansLI we should incorporate this https://doi.org/10.18122/redi_data.1.boisestate

@dblodgett-usgs if you're able to help her understand this repo or come up with a different way to systematically add things like this while I'm on leave that would be cooool.

bonus: we can try to address these, don't know how good the lat/lons are

**Comment by dblodgett-usgs** [comment]
Absolutely. @ewiggansLI -- hit me up on slack or here and we'll work through questions as they arise.

**Comment by dblodgett-usgs** [comment]
Leaving this as a WIP in https://github.com/internetofwater/ref_gages/pull/31
